### PR TITLE
Update trust_key_mng.md

### PR DESCRIPTION
**Pull request by scjane**

### CODE REVIEW

File: engine/security/trust/trust_key_mng.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ title: Manage keys for content trust
...
-| snapshot         | This key signs the current collection of image tags, preventing mix and match attacks.
-| timestamp        | This key allows Docker image repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side. |
-| delegation       | Delegation keys are optional tagging keys and allow you to delegate signing image tags to other publishers without having to share your targets key. |
+| Key        | Description                                                                                                                                                                                                                         |
+|:-----------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| root key   | Root of content trust for an image tag. When content trust is enabled, you create the root key once. Also known as the offline key, because it should be kept offline.                                                              |
```
Comment (mstanleyjones) [inline engine/security/trust/trust_key_mng.md]: I think you inadvertently added a lot of whitespace to the ends of these lines.

File: engine/security/trust/trust_key_mng.md [hunk 2]
```diff
@@ -37,9 +37,9 @@ repositories created with newer versions of Docker.
 ## Choosing a passphrase
 
 The passphrases you chose for both the root key and your repository key should
-be randomly generated and stored in a password manager.  Having the repository key
+be randomly generated and stored in a password manager. Having the repository key
```
Comment (mstanleyjones) [inline engine/security/trust/trust_key_mng.md]: 👍

File: engine/security/trust/trust_key_mng.md [hunk 3]
```diff
@@ -37,9 +37,9 @@ repositories created with newer versions of Docker.
...
 allows users to sign image tags on a repository. Passphrases are used to encrypt
-your keys at rest and ensures that a lost laptop or an unintended backup doesn't
+your keys at rest and ensure that a lost laptop or an unintended backup doesn't
```
Comment (mstanleyjones) [inline engine/security/trust/trust_key_mng.md]: 👍

Comment (scjane) [inline engine/security/trust/trust_key_mng.md]: Thanks :)

**Pull request merged**